x/ibc/02-client/handler.go

```diff
@@ -42,21 +59,26 @@ func HandleMsgCreateClient(ctx sdk.Context, k Keeper, msg MsgCreateClient) (*sdk
 }
 
 // HandleMsgUpdateClient defines the sdk.Handler for MsgUpdateClient
-func HandleMsgUpdateClient(ctx sdk.Context, k Keeper, msg MsgUpdateClient) (*sdk.Result, error) {
-	if err := k.UpdateClient(ctx, msg.ClientID, msg.Header); err != nil {
+func HandleMsgUpdateClient(ctx sdk.Context, k Keeper, msg exported.MsgUpdateClient) (*sdk.Result, error) {
+	if err := k.UpdateClient(ctx, msg.GetClientID(), msg.GetHeader()); err != nil {
 		return nil, err
 	}
 
+	attributes := make([]sdk.Attribute, len(msg.GetSigners())+1)
+	attributes[0] = sdk.NewAttribute(sdk.AttributeKeyModule, AttributeValueCategory)
+	for i, signer := range msg.GetSigners() {
```

Comment:
Same question as to whether there would ever be more than 1 signer.

Reply:
ditto

Reply:
ditto on fine then